lib/bandit/pipeline.ex

```diff
@@ -90,7 +90,7 @@ defmodule Bandit.Pipeline do
        ),
        do: {:ok, to_string(host), port || determine_local_port(local_info)}
 
-  @spec determine_local_port(ThousandIsland.Transport.socket_info()) :: integer()
+  @spec determine_local_port(ThousandIsland.Transport.socket_info()) :: :inet.port_number()
```

Comment:
Since Bandit is designed for `Plug` should it be `Plug.Conn.port_number()`?
I know it's the same thing in the end, so it is of little importance, but thought I'd ask anyway.

Reply:
Good point. Just updated the PR.
Thanks!